# Message for a detected particle
int32 particle_id
string material_type
float64 distance
float64 particle_x
float64 particle_y
float64 particle_z
float64 robot_x
float64 robot_y
float64 robot_z
float64 detection_timestamp
